.travis.yml

```diff
@@ -9,8 +9,12 @@ python:
   - 3.6
   - pypy
 
-before_install:
-  - pip install Cython
+# workaround for https://github.com/travis-ci/travis-ci/issues/9815
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```

Comment:
Since the official Travis Xenial release, this workaround is no longer necessary:

https://blog.travis-ci.com/2018-11-08-xenial-release

Reply:
I tried to remove it, but Travis fails:
```
3.7 is not installed; attempting download
Downloading archive: https://s3.amazonaws.com/travis-python-archives/binaries/ubuntu/14.04/x86_64/python-3.7.tar.bz2
$ curl -sSf -o python-3.7.tar.bz2 ${archive_url}
curl: (22) The requested URL returned error: 403 Forbidden
Unable to download 3.7 archive. The archive may not exist. Please consider a different version.
```

https://travis-ci.org/snowballstem/pystemmer/jobs/465741265

Looks like we still need this.

Reply:
Sorry, I should be more precise. You can drop the `sudo` line. The `dist: xenial` needs to remain.

Reply:
thank you!